tests/data/test_datagen_jobs.py

```diff
@@ -367,3 +370,578 @@ def test_supercell_check(mp_1200830, memory_jobstore):
+                                                'SYMMOPS': '1-4',
+                                                'NFORM': '500',
+                                                'MINSEP': '2.0',
+                                                'SLACK': 0.25,
+                                                'OVERLAP': 0.1,
+                                                'SYSTEM': 'Cubi'
+                                                },
+                              fragment_file=os.path.join(f'{test_dir}/data', 'h2o.xyz'),
+                              fragment_numbers=None,
+                              remove_tmp_files=True,
+                              num_processes=4).make()
+    
+    _ = run_locally(job, ensure_success=True, create_folders=True, store=memory_jobstore)
+    ats = read(job.output.resolve(memory_jobstore), index=":")
+    assert len(ats) == 4 and np.all(ats[0].positions[0] != ats[0].positions[1])
+
+
+def test_output_from_cell_seed(test_dir, memory_jobstore, clean_dir):
+    from ase.io import read
+    from pathlib import Path
+    import shutil
+    test_files_dir = test_dir / "data/SiO2.cell"
+    job = RandomizedStructure(struct_number=3,
+                              cell_seed_path=test_files_dir,
+                              num_processes=3).make()
+    
+    responses = run_locally(job, ensure_success=True, create_folders=True, store=memory_jobstore)
+    assert len(read(job.output.resolve(memory_jobstore),index=":")) == 3
+
+
+def test_build_multi_randomized_structure(memory_jobstore, clean_dir):
+    from autoplex.data.rss.flows import BuildMultiRandomizedStructure
+    from autoplex.data.common.utils import flatten
+    from pathlib import Path
+    import shutil
+    bcur_params={'soap_paras': {'l_max': 3,
+                                'n_max': 3,
+                                'atom_sigma': 0.5,
+                                'cutoff': 4.0,
+                                'cutoff_transition_width': 1.0,
+                                'zeta': 4.0,
+                                'average': True,
+                                'species': True,
+                                },
+                }
+    generate_structure = BuildMultiRandomizedStructure(tag="Si",
+        generated_struct_numbers=[50,50],
+        buildcell_options=[{'VARVOL': 20, 
+                            'VARVOL_RANGE': '0.75 1.25',
+                            'NATOM': '{6,8,10,12,14,16,18,20,22,24}',
+                            'NFORM': '1'}, 
+                           {'SYMMOPS':'1-2',
+                            'NATOM': '{7,9,11,13,15,17,19,21,23}',
+                            'NFORM': '1'}],
+        num_processes=8,
+        initial_selection_enabled=True,
+        selected_struct_numbers=[8,2],
+        bcur_params=bcur_params,
+        random_seed=None).make()
+
+    job = Flow(generate_structure, output=generate_structure.output) 
```

Comment:
same suggestion for more descriptive job names for all of the other job names as well